pulsar-websocket/src/main/java/org/apache/pulsar/websocket/AbstractWebSocketHandler.java

```diff
@@ -55,44 +56,56 @@ public AbstractWebSocketHandler(WebSocketService service, HttpServletRequest req
         request.getParameterMap().forEach((key, values) -> {
             queryParams.put(key, values[0]);
         });
-    }
 
-    @Override
-    public void onWebSocketConnect(Session session) {
-        super.onWebSocketConnect(session);
-        log.info("[{}] New WebSocket session on topic {}", session.getRemoteAddress(), topic);
+        checkAuth(response);
+    }
 
+    private void checkAuth(ServletUpgradeResponse response) {
         String authRole = "<none>";
         if (service.isAuthenticationEnabled()) {
             try {
                 authRole = service.getAuthenticationService().authenticateHttpRequest(request);
-                log.info("[{}] Authenticated WebSocket client {} on topic {}", session.getRemoteAddress(), authRole,
+                log.info("[{}] Authenticated WebSocket client {} on topic {}", request.getRemoteAddr(), authRole,
```

Comment:
I think that `session.getRemoteAddress().toString()` returns IP and port (such as `/172.21.205.28:58954`) but `request.getRemoteAddr()` returns only IP (such as `172.21.205.28`).
Is this difference acceptable?

Reply:
We can still printing the port by doing `request.getRemotePort()`. I'll fix these